contracts/interchain-token-service/src/msg.rs

```diff
@@ -83,6 +94,17 @@ pub enum QueryMsg {
     #[returns(HashMap<ChainNameRaw, Address>)]
     AllItsContracts,
 
+    /// Query all chain configs with optional frozen filter
+    // The list is paginated by:
+    // - start_after: the chain name to start after, which the next page of results should start.
+    // - limit: limit the number of chains returned, default is u32::MAX.
+    #[returns(Vec<ChainConfigResponse>)]
+    ItsChains {
+        filter: Option<ChainFilter>,
+        start_after: Option<ChainNameRaw>,
+        limit: Option<u32>,
```

Comment:
```suggestion
        #[serde(default = "default_pagination_limit"]
        limit: u32,
```

with 
```
const fn default_pagination_limit() -> u32{
    DEFAULT_LIMIT
}
```

this makes it clearer that we never do an unbounded search